.coderabbit.yaml

```
@@ -0,0 +1,95 @@
+# .coderabbit.yaml
+# CodeRabbit 설정 파일
+
+# 리뷰 설정
+reviews:
+  # 자동 리뷰 활성화
+  auto_review: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)